org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -285,24 +285,32 @@ VarRefOrModeTransition returns VarRef:
 
 Assignment:
     (lhs=[Parameter] (
-        (equals='=' rhs+=Value)
+        (equals='=' rhs+=Expression)
         | ((equals='=')? (
-            parens+='(' (rhs+=Value (',' rhs+=Value)*)? parens+=')'
-            | braces+='{' (rhs+=Value (',' rhs+=Value)*)? braces+='}'))
+            parens+='(' (rhs+=Expression (',' rhs+=Expression)*)? parens+=')'
+            | braces+='{' (rhs+=Expression (',' rhs+=Expression)*)? braces+='}'))
     ));
 
 /**
  * Parameter declaration with optional type and mandatory initialization.
  */
 Parameter:
     name=ID (':' (type=Type))?
-    ((parens+='(' (init+=Value (','  init+=Value)*)? parens+=')')
-        | (braces+='{' (init+=Value (','  init+=Value)*)? braces+='}')
+    ((parens+='(' (init+=Expression (','  init+=Expression)*)? parens+=')')
+        | (braces+='{' (init+=Expression (','  init+=Expression)*)? braces+='}')
     )?
 ;
 
-Value:
-    (parameter=[Parameter] | time=Time | literal=Literal | code=Code);
+Expression:
```

Comment:
I'm confused by this. I would expect an Expression to somehow different from a Value.  This looks to me the same as what we used to call a Value.

Reply:
I think the idea here is that @cmnrd plans to augment this rule later on. This PR is just meant to set the stage for that.